doc/hacking.rst

```diff
@@ -26,12 +26,16 @@ invocation of ``ocamlopt`` or ``ocamlc`` via the ``bootstrap.ml`` OCaml script.
 and uses this binary to build everything else. As a convenience, ``dune.exe``
 at the root of the source tree executes this binary.
 
-``$ make dev`` takes care of bootstrapping if needed, but if you want to just
-run the bootstrapping step itself, build the ``dune.exe`` target with
+Running:
+```
+make dev
+```
+takes care of bootstrapping if needed, but if you want to just run the
```

Comment:
What does "takes care of bootstrapping if needed" mean concretely?

Here is a (partially evaluated) guess:
* It builds `_boot/dune.exe`.
* It uses `_boot/dune.exe` to build "everything else" from the above paragraph. What does "everything else" include?

Reply:
We should really clarify that this is doing `./dune.exe build @install`. So everything else, means all the packages and importantly the dune binary.